lib/authentication.py

```diff
@@ -3,17 +3,24 @@
 import requests
 
 
+def find_candidate_by_id(json_candidates, candidate_id):
```

Comment:
Not sure this is the right place for this function... I would probably create a separate utils.py for it  ;)